psi4/driver/p4util/fcidump.py

```diff
@@ -248,8 +249,71 @@ def compare_fcidumps(expected, computed, digits, label):
     if header_diff:
         message = ("\tComputed FCIDUMP file header does not match expected header.\n")
         raise TestComparisonError(header_diff)
-    # Compare integrals
-    compare_arrays(ref_intdump, intdump, digits, label)
+    success(label)
+
+    nbf = header['norb']
+    # Skip the last line, i.e. Enuc + Efzc
+    ints = intdump[:-1, 0]
+    # Get indices
+    idxs = intdump[:, 1:].astype(np.int) - 1
+    # Slices
+    sl = slice(ints.shape[0] - nbf, ints.shape[0])
+    # Extract orbital energies
+    epsilon = np.zeros(nbf)
+    epsilon[idxs[sl, 0]] = ints[sl]
+    # Count how many 2-index integrals we have
+    sl = slice(sl.start - nbf * nbf, sl.stop - nbf)
+    two_index = np.all(idxs[sl, 2:] == -1, axis=1).sum()
+    sl = slice(sl.stop - two_index, sl.stop)
+    # Extract Hcore
+    Hcore = np.zeros((nbf, nbf))
+    Hcore[(idxs[sl, 0], idxs[sl, 1])] = ints[sl]
+    Hcore[(idxs[sl, 1], idxs[sl, 0])] = ints[sl]
+    # Extract ERIs
```

Comment:
Can you add a newline above the comment lines?

Reply:
Done.